src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/TaskListService.kt

```diff
@@ -223,7 +223,7 @@ class TaskListService(
 		if (tasks.containsKey(key)) {
 
 			loggedTaskStates[key] = EventTypes.FAILURE
-			metrics.setTasksGivenUpOn(1.0)
+			metrics.setTasksGivenUpOn(loggedTaskStates.values.filter{v -> v == EventTypes.FAILURE}.size)
```

Comment:
Trengs denne rader her eller ville det vart nok med å ha den i `updateTaskState()`?

Reply:
Trenger den for å få test til å fungere. Dette henger sammen med at updateTaskState() ikke kalles når state endres til failure